### PR TITLE
Updated Plugins

### PR DESCRIPTION
**Pull request by fdcastel**
Alternative to (now defunct) #94.

Also, I'm adding BitwiseWrap plugin (discussed in #74). Maybe it could be used it in a future plugin tutorial? (it's simpler than the NUnit one).

Please note the build script is changed to build all three plugins (NUnit, VSTests and Bitwise). If this is not ok for any reason, just discard this change ;)

### CODE REVIEW

File: src/plugins/Fable.Plugins.BitwiseWrap.fsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+namespace Fable.Plugins
+
+#r "../../build/fable/bin/Fable.exe"
+
+open Fable.AST
+open Fable.AST.Fable
+open Fable.FSharp2Fable
+
+type RandomPlugin() =
```
Comment (alfonsogarciacaro) [inline src/plugins/Fable.Plugins.BitwiseWrap.fsx]: Can you please change the name of this plugin? :)

**Comment by alfonsogarciacaro** [comment]
Fantastic work, thanks a lot for that! With a small change you've made the Test plugin much more powerful :)

Yes, sorry, I was waiting to merge the `imports` branch before accepting the PR to avoid conflicts. I wrote a small comment, could you please fix it? I'll merge the PR right away :+1:

File: src/plugins/Fable.Plugins.BitwiseWrap.fsx (hunk 1 above)
Comment (fdcastel) [inline src/plugins/Fable.Plugins.BitwiseWrap.fsx]: DOH! :sweat_smile: 

Fixing it...

**Comment by fdcastel** [comment]
Done!

**Pull request merged**